components/admin/requests/physician-assessment/Card.tsx

```diff
@@ -108,6 +168,23 @@ const Card: FC<Props> = props => {
               </GridItem>
             </>
           )}
+          {/* NOTETOSELF: I think this is conditionally rendered? Will ask designers */}
+          <GridItem>
+            <Text as="p" textStyle="body-regular" textAlign="left">
+              Mobility aids
+            </Text>
+          </GridItem>
+          <GridItem>
+            <List>
+              {mobilityAids?.map(aid => (
+                <ListItem key={aid}>
+                  <Text as="p" textStyle="body-regular" textAlign="left">
+                    {titlecase(aid)}
+                  </Text>
+                </ListItem>
+              ))}
+            </List>
+          </GridItem>
```

Comment:
I must have missed this when building the card. It's in some of the specs and missing in others, I'll confirm with designers if this section is conditionally rendered

Reply:
Resolved!